src/Configs/Properties.vala

```diff
@@ -27,9 +27,9 @@ namespace Quotes.Configs {
      */
     public class Properties {
 
-        public abstract const string TITLE_HEADER_BAR = "Quotes";
-        public abstract const string CLOSE = "Close";
-        public abstract const string WELCOME = "Welcome";
+        public abstract const string TITLE_HEADER_BAR = (_("Quotes"));
```

Comment:
```suggestion
        public abstract const string TITLE_HEADER_BAR = _("Quotes");
```

Although this works as expected, you don't need the outside brackets.

---

If you want to make an argument string in a method like this:

```vala
// Create a HeaderBar for an app
var headerbar = new Gtk.HeaderBar ();
// Setting title
headerbar.set_title ("My App");
```

In this case, you have to envelop the string `"My App"` with `_(` and `)`, so it will look like:

```vala
headerbar.set_title (_("My App"));
```

The outside brackets is grammatically necessary to tell the compiler that this sentence is a method. The inside brackets is what we just added to make the string translatable.

However, if you want to make a string in a substitution sentence (`=`) like this:

```vala
// Create a HeaderBar for an app
var headerbar = new Gtk.HeaderBar ();
// Setting title
headerbar.title = "My App";
```

In this case, you have to envelop the string `"My App"` with `_(` and `)` as same as you do in the previous example, so it will look like:

```vala
// Create a HeaderBar for an app
var headerbar = new Gtk.HeaderBar ();
// Setting title
headerbar.title = _("My App");
```